internal/config/config.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"flag"
+
+	"github.com/Stern-Ritter/metrics-and-alerting-service/internal/utils"
+)
+
+type AgentConfig struct {
+	SendMetricsURL        string `env:"ADDRESS"`
+	SendMetricsEndPoint   string
+	UpdateMetricsInterval int `env:"POLL_INTERVAL"`
+	SendMetricsInterval   int `env:"REPORT_INTERVAL"`
+}
+
+func (c *AgentConfig) ParseFlags() error {
+	flag.StringVar(&c.SendMetricsURL, "a", "localhost:8080", "address and port to run server in format <host>:<port>")
```

Comment:
CHANGE REQUEST: this is an application specific code.

Reply:
done